drivers/sensor/stm32_temp/stm32_temp.c

```diff
@@ -88,7 +88,20 @@ static int stm32_temp_channel_get(const struct device *dev, enum sensor_channel
 	}
 
 #if HAS_CALIBRATION
-#if defined(CONFIG_SOC_SERIES_STM32U5X)
+
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+	/*
```

Comment:
Please check that in details for stm32h7 devices: stm32h723,  stm32h735, stm32h7b3 etc. There are many ADC version in this soc family.